src/index.js

```diff
@@ -52,7 +53,15 @@ app.use(apiRoutes);
 app.get('/', (req, res) => {
   res.send("Welcome to Barefoot Nomad Endpoints' Page");
 });
-
+app.post('/signup', (req, res) => {
+  const token = Authentication.signJwt({ user_id: 2345, is_admin: false });
```

Comment:
is there a reason why the user object is hardcoded here?

Reply:
I actually don't have the signup route in my branch and wanted to test if a token is returned. I don't know if I should remove it.

Reply:
@okabamac That means this PR is dependent on the signup feature. We can hold off till that PR is merged then

Reply:
OK, no problem. I will wait. Thanks.